packages/sds/src/message_channel/repair/utils.ts

```diff
@@ -0,0 +1,78 @@
+import { sha256 } from "@noble/hashes/sha2";
+import { bytesToHex } from "@noble/hashes/utils";
```

Comment:
```suggestion
import { bytesToHex } from "@waku/utils/bytes";
```

Best to use the same one across the codebase.